subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/zephyr_img_mgmt.c

```diff
@@ -80,24 +75,40 @@ zephyr_img_mgmt_flash_check_empty(uint8_t fa_id, bool *out_empty)
 
 		rc = flash_area_read(fa, addr, data, bytes_to_read);
 		if (rc != 0) {
-			flash_area_close(fa);
 			return MGMT_ERR_EUNKNOWN;
 		}
 
 		for (i = 0; i < bytes_to_read / 4; i++) {
 			if (data[i] != erased_val_32) {
 				*out_empty = false;
-				flash_area_close(fa);
 				return 0;
 			}
 		}
 	}
 
 	*out_empty = true;
-	flash_area_close(fa);
+
 	return 0;
 }
 
+static int
+zephyr_img_mgmt_flash_check_empty(uint8_t fa_id, bool *out_empty)
```

Comment:
Drop the zephyr_ prefix here, this is internal function and will not collide with anything else, and we do not support other systems so there is no point in making the function Zephyr specific.
We will weed out the zephyr_ prefix in other functions later.

Reply:
Fixed